plugins/inputs/snmp/README.md

```diff
@@ -141,8 +135,7 @@ option operate similar to the `snmpget` utility.
 
 #### Table
 
-Use a `table` to configure the collection of a SNMP table.  SNMP requests
-formed with this option operate similarly way to the `snmptable` command.
+Use a `table` to configure the collection of a SNMP table.
```

Comment:
Same comment here, please keep the extra clarity.

Reply:
Good catch!